src/index.html

```diff
@@ -6,11 +6,56 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      action="/"
+      method="get"
+      class="search-bar "
+    >
+      <label>
+        <img
+          src="images/Search.svg"
+          alt="icon search"
+          class="icon"
+        >
+        <input
+          type="search"
+          class="search"
+          data-qa="keypress"
+          placeholder="Try &ldquo;Los Angeles&rdquo;"
+        >
+      </label>
+    </form>
+
+    <form
+      data-qa="small"
+      action="/"
+      method="get"
+      class="search-bar small"
+    >
+      <label>
+        <img
+          src="images/Search.svg"
+          alt="icon search"
+          class="icon small"
+        >
+        <input
+          type="search"
+          class="search small"
```

Comment:
```suggestion
          class="search search--small"
```
![image](https://user-images.githubusercontent.com/55200530/225913546-37abd5cf-db1a-4e4f-950f-d8be73bcea17.png)